src/software/ai/hl/stp/play/corner_kick_play.h

```diff
@@ -22,11 +22,11 @@ class CornerKickPlay : public Play
 
     void getNextTactics(TacticCoroutine::push_type &yield) override;
 
-   private:
     // The maximum distance from the corner that the ball can be for it to be
     // considered a corner kick
     static constexpr double BALL_IN_CORNER_RADIUS = 0.5;
 
+   private:
```

Comment:
why was this moved?

Reply:
The indirect free kick play needed access to the `BALL_IN_CORNER_RADIUS`